### ISSUE TITLE
Hash Collision for 0xDCAE6935

### ISSUE THREAD

**Issue by TheRouletteBoi** [TheRouletteBoi/rdr-nativedb-data#6]
@emoose 

Hash 0xDCAE6935 has a conflict. It is related to `rdrSharedTextureManager::LoadFragmentDictionaryList`. So ~RELOAD_SMICTIONARY_LIST~ does not match with name

**Comment by emoose** [comment]
Yeah it looked weird to me too, but if you look in rdrSharedTextureManager::LoadFragmentDictionaryList it does make use of a "$/tune/settings/smictofragmap" string near the start of it.

I'm not actually sure where I got the smictionary word when bruting it, that string seems the only mention of it

**Comment by TheRouletteBoi** [comment]
> Yeah it looked weird to me too, but if you look in rdrSharedTextureManager::LoadFragmentDictionaryList it does make use of a "$/tune/settings/smictofragmap" string near the start of it.
> 
> I'm not actually sure where I got the smictionary word when bruting it, that string seems the only mention of it E: ah nvm there is a `sagWorldScript::scrRELOAD_SMICTIONARY_LIST` func symbol which I guess it came from, that func has the same call to `rdrSharedTextureManager::LoadFragmentDictionaryList` inside it

After some verification this turned out to be true
